proctor-common/src/main/java/com/indeed/proctor/common/model/ConsumableTestDefinition.java

```diff
@@ -31,6 +31,7 @@ public class ConsumableTestDefinition {
 
     /** @see TestDefinition#getDependsOn() */
     @Nullable private TestDependency dependsOn;
+    private boolean isDynamic = false;
```

Comment:
This field needs to be added the normal test definition class also. See similarly done for enableUnitlessAllocations 
 in https://github.com/indeedeng/proctor/pull/130/files#diff-bc7f872a1a8cc43b096c242131486807b86d8d252809445bbd61a5bc5e853ae7